packages/test/test-driver-definitions/src/index.ts

```diff
@@ -3,13 +3,14 @@
  * Licensed under the MIT License.
  */
 
-import { ITestDriver } from "./interfaces";
+import { ITestDriver, ITelemetryBufferedLogger } from "./interfaces";
 
 declare global {
-    /**
-     * This function needs to be provided by the environment leverging these defintions, like a mocha test hook.
-     */
+    /** This function needs to be provided by the environment leveraging these defintions, like a mocha test hook. */
     export function getFluidTestDriver(): ITestDriver;
+
+    /** This function may be provided by the environment, like a mocha test hook */
+    export function getTestLogger(singleton?: boolean): ITelemetryBufferedLogger;
```

Comment:
why do we have to choose if a singleton or not?

Reply:
looks to me like this can always be a singleton, and we can get rid of the parameter

Reply:
Hmm that's true, thanks!